network-api/networkapi/buyersguide/pagemodels/product_category.py

```diff
@@ -48,16 +47,11 @@ class BuyersGuideProductCategory(models.Model):
         blank=True,
     )
 
-    @property
-    def published_product_count(self):
-        # TODO: REMOVE: LEGACY FUNCTION
-        return Product.objects.filter(product_category=self, draft=False).count()
-
```

Comment:
let's put this back in for now

```suggestion
    @property
    def published_product_count(self):
        # TODO: REMOVE: LEGACY FUNCTION
        return Product.objects.filter(product_category=self, draft=False).count()

```